services/movie.py

```diff
@@ -0,0 +1,38 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet:
+    query_set_movies = Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        query_set_movies = query_set_movies.filter(genres__id__in=genres_ids,
+                                                   actors__id__in=actors_ids)
+    if genres_ids:
+        query_set_movies = query_set_movies.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        query_set_movies = query_set_movies.filter(actors__id__in=actors_ids)
```

Comment:
These conditions will further filter the queryset, but if both `genres_ids` and `actors_ids` are provided, the first condition will already have filtered the queryset. Consider revising the logic if the intention is to filter by either genres or actors independently.